docker/chronicle.yaml

```diff
@@ -99,16 +106,18 @@ services:
 
   chronicle-sawtooth-api:
     environment:
-      - *rustbacktrace
-      - *rustlog
+      - RUST_BACKTRACE=full
+      - RUST_LOG=info,cranelift_codegen=off,wasmtime_cranelift=off,regalloc2::ion=off,cranelift_wasm=off
       - PGHOST=chronicle-pg
       - PGPASSWORD=chronicle
     image: &chronicleImage chronicle-amd64:${ISOLATION_ID:-local}
     entrypoint: /bin/bash
+    links:
+      - "opa-init:opa-init"
     command:
       - -exc
       - |
-        sleep 10
+        sleep 60
```

Comment:
this one would be better as a `wait-for-it` and then *maybe* an extra sleep.  I thought we had already put that in somewhere, no?

Reply:
We don't bake wait for it into the chronicle image, would need a derived container or test dependencies in released artefacts